frontend/src/pull.ts

```diff
@@ -187,6 +187,10 @@ export class Pull extends PullData {
     const linkIdPrefix = isReview ? "pullrequestreview" : "issuecomment";
     return this.url() + `#${linkIdPrefix}-${sig.data.comment_id}`;
   }
+
+  linkToBranch() {
+    return "https://github.com/" + this.repo + "/tree/" + this.base.ref;
```

Comment:
This will generate a link like: https://github.com/ifixit/ifixit/tree/shopify-cancellation-tweaking

Which *does* have a link to the relevant pull: 
![image](https://user-images.githubusercontent.com/26855/226439052-cc2f5d31-e671-4da4-821f-852ea9917bc6.png)

But I wonder if this url might be better:
https://github.com/ifixit/ifixit/compare/shopify-cancellation-tweaking

![image](https://user-images.githubusercontent.com/26855/226439225-7426f691-e917-430e-8da2-9a1f254276ea.png)

Thoughts?

Reply:
That's a good idea, I think the link to the pull request in the branch view is a bit more difficult to spot